pipreqs/pipreqs.py

```diff
@@ -102,13 +122,26 @@ def get_all_imports(path, encoding=None, extra_ignore_dirs=None, follow_links=Tr
         dirs[:] = [d for d in dirs if d not in ignore_dirs]
 
         candidates.append(os.path.basename(root))
-        files = [fn for fn in files if os.path.splitext(fn)[1] == ".py"]
+        if PythonExporter and not ignore_notebooks:
+            files = [fn for fn in files if filter_ext(fn, [".py", ".ipynb"])]
+        else:
+            files = [fn for fn in files if filter_ext(fn, [".py"])]
```

Comment:
```suggestion
            extensions =  get_file_extensions()
            files = [fn for fn in files if filter_ext(fn, extensions)]
```

Reply:
> Decided to not implement this function because it makes the file extensions used by the program "hidden". The user will only know the file extension being used in the program if the user reads the function directly, making our code a little bit less readable

^^